### DISCUSSION TITLE
showing account activity in modal

### DISCUSSION THREAD

**Discussion by ethdev279** [WalletConnect/walletconnect-monorepo#4038] [Developer Support] [answered]
Guys, how can I configure web3modal to show account activity like this?
![Screenshot 2023-12-11 at 8 51 02 PM](https://github.com/WalletConnect/walletconnect-monorepo/assets/45661693/9d5c4ef0-6783-4ccd-802c-fffa3784d569)

**Comment by glitch-txs** [comment, marked as answer]
Hi, this is shown by default, make sure to be in the latest version
